applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -567,7 +562,6 @@ static void datamode_quit(struct k_work *work)
 	} else {
 		LOG_WRN("missing datamode handler");
 	}
-	(void)exit_datamode(DATAMODE_EXIT_OK);
```

Comment:
Should this line not be deleted? With is removed I don't get the unsolicited `#XDATAMODE: 0` when exiting datamode, at least when exiting datamode for the Socket AT commands. I added the line back and things work as expected. For reference though, I back ported this one commit on top of the v2.0.0 release, is it possible that is causing the issue?

Reply:
It should not, I have it in my local code.
Looks like a mistake in PR submission.

Reply:
Yes it should be there like "(void)exit_datamode(0);"
It's a mistake in PR submission, that it does exist in my local code.